### ISSUE TITLE
panicked at 'assertion failed: !module_name_parts.is_empty()', src/run.rs:569:5

### ISSUE THREAD

**Issue by harrysarson** [mpizenberg/elm-test-rs#63]
To reproduce run `elm-test-rs src` in a directory that looks like this:

```
❯ ls -R                                                                                                                                                   [18:34:02]
.:
elm.json  src

./src:
Something.elm
```

output:

```
❯ elm-test-rs src                                                                                                                                         [18:41:20]

elm-test-rs 1.0.0-alpha for elm 0.19.1
--------------------------------

thread 'main' panicked at 'assertion failed: !module_name_parts.is_empty()', src/run.rs:564:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
---
 - Found based on [this](https://github.com/rtfeldman/node-test-runner/tree/0.19.1-revision6/example-application-src) from the node test runner repo.
 - Tested with 3f6e0a8.

**Comment by mpizenberg** [comment]
damn, thanks, I'll look into it!

**Comment by mpizenberg** [comment]
Should be fixed with e23ba0f
